app/helpers/feed_helper.rb

```diff
@@ -14,4 +15,61 @@ def cached_feed(url)
   def feed_entry_thumbnail_url(entry)
     FeedEntryImage.new(entry).thumbnail_url
   end
+
+  def tumblr_feed_content(landing_page, options = {})
```

Comment:
Why do we have this method? It's only called when the landing page layout is "default", but we only have a Tumblr feed for the fashion collection landing page which uses the "browse" layout.